tests/foreman/cli/test_activationkey.py

```diff
@@ -58,173 +56,167 @@ def setUp(self):
 
     def _make_activation_key(self, options=None):
         """Make a new activation key and assert its success"""
-
         if options is None:
             options = {}
 
         # Use default organization if None are provided
-        if (
-                not options.get('organization', None)
-                and not options.get('organization-label', None)
-                and not options.get('organization-id', None)):
+        if (not options.get('organization', None)
+            and not options.get('organization-label', None)
+            and not options.get('organization-id', None)):
             options['organization-id'] = self.org['id']
 
         # Create activation key
         ackey = make_activation_key(options)
 
         # Fetch it
-        result = ActivationKey.info(
-            {
-                'id': ackey['id']
-            }
-        )
+        result = ActivationKey.info({'id': ackey['id']})
 
         self.assertEqual(
             result.return_code,
             0,
-            "Activation key was not found: %s" % str(result.stderr))
+            'Activation key was not found: {}'.format(result.stderr))
```

Comment:
i wasn't sure if it was this way or the other. i can put the 0 in though. i think that was the last conclusion